client/client.go

```diff
@@ -227,6 +227,7 @@ func (client *Client) do(method, path string, query url.Values, headers map[stri
 
 	if v != nil {
 		dec := json.NewDecoder(rsp.Body)
+		dec.UseNumber()
```

Comment:
Can we have a unit test that breaks without those changes?  Otherwise this is likely to be dropped at some point and we won't notice.